lib/Doctrine/ODM/MongoDB/Query/Query.php

```diff
@@ -344,16 +359,16 @@ static function ($value) {
      *
      * Note: while this method could strictly take a MongoDB\Driver\Cursor, we
      * accept Traversable for testing purposes since Cursor cannot be mocked.
-     * HydratingIterator and CachingIterator both expect a Traversable so this
-     * should not have any adverse effects.
+     * HydratingIterator, CachingIterator and BaseIterator expect a Traversable 
```

Comment:
Please use an [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) here:
```suggestion
     * HydratingIterator, CachingIterator, and BaseIterator expect a Traversable 
```

Reply:
Applied in 87fc9210df0a412b36be475e3e9f7b925aa8201a